main.js

```diff
@@ -1 +1,14 @@
+function getButton() {
+  let buttons = document.querySelectorAll(".btn");
+  console.log(buttons);
+  let display = document.querySelector("#displayScreen");
+  for (let button of buttons) {
+    console.log(button.textContent);
+    button.addEventListener("click", function () {
+      alert(buttons);
+    })
+      ;
+  }
 
+}
+getButton();
```

Comment:
Great work on this Racq! I love how you took that extra step to work with the for loop and trying to figure out how things are working using the console.log and the alert. These things will take you a long way as a programmer. Keep on pushing!